src/main.js

```diff
@@ -41,31 +42,52 @@ function displayOff() {
 
 /////////////// iteration 2 ///////////////
 
-function createCard(title, idea) {
+function createCard() {
   var title = inputTitle.value;
   var body = inputBody.value;
   if (!(title === '' || body === '')) {
     var userCard = new Idea(title, body);
     ideas.push(userCard);
-    sectionIdeaCards.insertAdjacentHTML('beforeend',
-      '<article class="card">
-        <section class="card-banner-top">
-          <img class="banner-star" src="assets/star.svg" alt="card-star-icon">
-          <img class="banner-x" src="assets/delete.svg" alt="card-delete-icon">
-        </section>
-        <section class="card-content">
-          <h2 class="card-title">Idea title</h2>
-          <h2 class="card-body">Idea body</h2>
-        </section>
-        <section class="card-footer">
-          <img src="assets/comment.svg" alt="icon-comment">
-          <h3 class="comment-text">Comment</h3>
-        </section>
-      </article>');
+    displayCard(title, body);
+  } 
+};
+
+function displayCard(title, body) {
+  clearTemplates();
+  var additionalCard;
+  for (var i = 0; i < ideas.length; i++) {
+    additionalCard =
+    `<article class="card">
+      <section class="card-banner-top">
+        <img class="banner-star" src="assets/star.svg" alt="card-star-icon">
+        <img class="banner-x" src="assets/delete.svg" alt="card-delete-icon">
+      </section>
+      <section class="card-content">
+        <h2 class="card-title">${ideas[i].title}</h2>
```

Comment:
Im still in awe of how simple this was. Essentially whenever the user adds a new title and body this is telling it to add to the new idea card to the next index on the page.